test/lib/jdk/test/lib/thread/VThreadPinner.java

```diff
@@ -94,6 +94,7 @@ private static void callback() {
      * Runs the given task on a virtual thread pinned to its carrier. If called from a
      * virtual thread then it invokes the task directly.
      */
+    @SuppressWarnings("unchecked")
```

Comment:
If you change the code to the follow, same in VThreadRunner then it would avoid the the SW on the method and avoid the other change you have to VThreadRunner.
```
            @SuppressWarnings("unchecked")
            var x = (X) ex;
            throw x;
```

Reply:
Thanks!